compose/README.md

```diff
@@ -37,6 +37,17 @@ At first you need to install docker with [compose](https://docs.docker.com/compo
 - [docker](https://docs.docker.com/installation/)
 - [docker-compose](https://docs.docker.com/compose/install/)
 
+**Currently there is an issue with dockers storage driver**
```

Comment:
Can you please re-formulate this to sound more like `if you encounter this and this try to change your storage driver`. As this is very dependent on your OS. I know that Ubuntu 14.04 had similar issues but does not have it today. So this is something very OS-specific and should not be a general recommendation.

Thanks @innovate-invent!

Reply:
Do you know what changed in 14.04?

Reply:
No, sorry.